source/quick-reference.txt

```diff
@@ -0,0 +1,561 @@
+              Address = new BsonDocument
+              {
+                  {"street", "Pizza St"},
+                  {"zipcode", "10003"},
+              },
+              Borough = "Manhattan",
+          };
+
+          return await _restaurantsCollection.ReplaceOneAsync(filter, newPizzaRestaurant);
+
+   * - | **Delete a Document**
+       |
+       | `API Documentation <{+api-root+}/Overload_MongoDB_Driver_IMongoCollectionExtensions_DeleteOne.htm>`__
+       | :ref:`Usage Example <csharp-delete-one>`
+       | :ref:`Fundamentals <csharp-delete-guide>`
+
+     - .. code-block:: csharp
+          :copyable: true
+
+          var filter = Builders<Restaurant>.Filter
+            .Eq(restaurant => restaurant.Name, "Ready Penny Inn");
+
+          return _restaurantsCollection.DeleteOne(filter);
+          
+   * - | **Delete a Document (Async)**
+       |
+       | `API Documentation <{+api-root+}/Overload_MongoDB_Driver_IMongoCollectionExtensions_DeleteOneAsync.htm>`__
+       | :ref:`Usage Example <csharp-delete-one>`
+       | :ref:`Fundamentals <csharp-delete-guide>`
+
+     - .. code-block:: csharp
+          :copyable: true
+
+          var filter = Builders<Restaurant>.Filter
+            .Eq(restaurant => restaurant.Name, "Ready Penny Inn");
+
+          return await _restaurantsCollection.DeleteOneAsync(filter);
+          
+   * - | **Delete Multiple Documents**
+       |
+       | `API Documentation <{+api-root+}/Overload_MongoDB_Driver_IMongoCollectionExtensions_DeleteMany.htm>`__ 
+       | :ref:`Usage Example <csharp-delete-many>`
+       | :ref:`Fundamentals <csharp-delete-guide>`
+
+     - .. code-block:: csharp
+          :copyable: true
+ 
+          var filter = Builders<Restaurant>.Filter
+            .Regex(restaurant => restaurant.Name, "Green");
+
+          return _restaurantsCollection.DeleteMany(filter);
+
+   * - | **Delete Multiple Documents (Async)**
+       |
+       | `API Documentation <{+api-root+}/Overload_MongoDB_Driver_IMongoCollectionExtensions_DeleteMany.htm>`__ 
+       | :ref:`Usage Example <csharp-delete-many>`
+       | :ref:`Fundamentals <csharp-delete-guide>`
+
+     - .. code-block:: csharp
+          :copyable: true
+ 
+          var filter = Builders<Restaurant>.Filter
+            .Regex(restaurant => restaurant.Name, "Green");
+
+          return await _restaurantsCollection.DeleteManyAsync(filter);
+
+   * - | **Access Data from a Cursor Iteratively**
+       |
+       | `API Documentation <{+api-root+}/T_MongoDB_Driver_IAsyncCursor_1.htm>`__
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: csharp
+
+             var filter = Builders<Restaurant>.Filter
+               .Eq(restaurant => restaurant.Cuisine, "Afghan";
+
+             var cursor = _restaurantsCollection.Find(filter).ToCursor(); 
+             cursor.MoveNext();
+             foreach (Restaurant r in cursor.Current)
+             {
+                 WriteLine(r.ToJson());
```

Comment:
Suggestion: I think we should show the full syntax here to avoid potential confusion for any that don't know about the `using static System.Console` line of code. 

```suggestion
                 Console.WriteLine(r.ToJson());
```